client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -124,21 +178,17 @@ def process(self, instance):
             return
 
         path = path.replace(".abc", "_proxy.abc")
+        args["file"] = path
         if not instance.data.get("includeParentHierarchy", True):
             # Set the root nodes if we don't want to include parents
             # The roots are to be considered the ones that are the actual
             # direct members of the set
-            options["root"] = instance.data["proxyRoots"]
+            args["root"] = instance.data["proxyRoots"]
 
         with suspended_refresh(suspend=suspend):
             with maintained_selection():
                 cmds.select(instance.data["proxy"])
-                extract_alembic(
-                    file=path,
-                    startFrame=start,
-                    endFrame=end,
-                    **options
-                )
+                extract_alembic(**args)
```

Comment:
You're passing the `args` as `**kwargs` (keyword arguments). As such, to avoid confusion I'd recommend renaming the variable accordingly to `kwargs` to match python conventions.